Documentation/ExtensionArchitecture/Extbase/Reference/Controller/ActionController.rst

```diff
@@ -120,6 +120,36 @@ displayed by the :php:`indexAction` of the :php:`BlogController`.
 
 .. include:: /CodeSnippets/Extbase/Controllers/ForwardAction.rst.txt
 
+You need to ensure that every target controller is listed as a valid
+target in the extension's `ext_localconf.php` file in the relevant
+:php:`ExtensionUtility::configurePlugin()` section. Otherwise, the object
```

Comment:
It is possible to add a plugin without calling the utility method. Not sure if  we should mention this, however this sentence does not seem 100% correct as configurePlugin is only a utility funciton and not mandatory